TestFoundation/TestNSString.swift

```diff
@@ -1056,7 +1056,13 @@ class TestNSString: LoopbackServerTest {
         do {
             let path: NSString =  "~/foo/bar/"
             let result = path.standardizingPath
+#if DARWIN_COMPATIBILITY_TESTS
+            // FIXME: Darwin gives unavailable for 'appendingPathComponent'
```

Comment:
This needs to be investigated as well.

Reply:
Filed https://bugs.swift.org/browse/SR-10916